src/fields/jsgrid.field.control.js

```diff
@@ -91,16 +93,16 @@
         },
 
         filterTemplate: function() {
-            var $result = this._createSearchButton();
+            var $result = this.searchButton ? this._createSearchButton() : $("");
```

Comment:
Initial idea behind was: If user needs filtering, he probably needs search button. If he doesn't need filtering, he can turn it off.
Do you have scenario when you need to hide search button? And what is about the button to clear filter? Is it stays visible on purpose?